doc/src/modules/physics/mechanics/autolev_parser.rst

```diff
@@ -0,0 +1,156 @@
+.. _autolev_parser:
+
+==============
+Autolev Parser
+==============
+
+.. role:: input(strong)
+
+Introduction
+============
+Autolev (now superseded by MotionGenesis) is a domain specific language
+used for symbolic multibody dynamics. The SymPy mechanics module now has
+enough power and functionality to be a fully featured symbolic dynamics
+module. This parser parses Autolev (version 4.1) code to SymPy code by making
+use of SymPy's math libraries and the mechanics module.
+
+The parser has been built using the `ANTLR <http://www.antlr.org/>`_ framework and its main purpose
+is to help former users of Autolev to get familiarized with multibody dynamics
+in SymPy.
+
+The sections below shall discuss details of the parser like usage, gotchas,
+issues and future improvements.
+For a detailed comparison of Autolev and SymPy/PyDy you might want to look at
+the PyDy for Autolev Users guide.
+
+.. _usage:
+
+Usage
+=====
+
+We first start with an Autolev code file.
+
+Let us take this example
+(Comments ``%`` have been included to show the Autolev responses):
+::
+
+    # % double_pendulum.al
```

Comment:
What are the `#` for?

Reply:
I think you should remove the leading hash and make use of the `..code::` directive, you may need to specify a null language to avoid syntax highlighting.